src/Diff/Internal/AliasGroupListPatcher.php

```diff
@@ -37,7 +36,7 @@ public function patchAliasGroupList( AliasGroupList $groups, Diff $patch ) {
 	}
 
 	/**
-	 * @see MapPatcher
+	 * @see Diff\Patcher\MapPatcher
```

Comment:
meh, I am still half-awake. Why removing the import here then? I don't know all stuff PHP does internally but having the "use" statement doesn't mean the class is loaded, it only gets loaded once used in code (ie. not in the documention). So this import wouldn't make any change regarding performance, memory usage etc as far as I get it (but maybe I am wrong).

I am not opposing to this change here. I am just a bit surprised as I've seen multiple places in Wikibase which import things just to have a simpler reference in the phpdoc to the class/its method, so I thought this is a general practice.